common.h

```diff
@@ -367,6 +367,8 @@ namespace fsgrids {
       int sysBoundaryLayer; /*!< System boundary layer index. */
       Real maxFsDt;         /*!< maximum timestep allowed in ordinary space by fieldsolver for this cell**/
       int fsGridRank;       /*!< Rank in the fsGrids cartesian coordinator */
+      uint SOLVE;           /*!< Bit mask to determine whether a given cell should solve E or B components. */
```

Comment:
Is this one a leftover from the PML code?
I guess this is not relevant for the filtering itself, right?

Reply:
These  is some conflict for common.h which has confused me. This is actually not from the PML code.

Reply:
That's our new way of doing @ursg, since the fluffiness-related Yee-fixing of boundaries.

Reply:
Oh, alright, true. Then this is all good.